src/isomorphic/classic/class/__tests__/ReactClass-test.js

```diff
@@ -58,6 +58,52 @@ describe('ReactClass-spec', function() {
       .toBe(propValidator);
   });
 
+    it('gives methods a displayName for testing purposes', function() {
```

Comment:
The lint fails here because of the indentation: https://travis-ci.org/facebook/react/jobs/120646658#L241-L273